docs/examples/plot_streamThickness.py

```diff
@@ -0,0 +1,83 @@
+"""
+Simple example to vary streamline thickness based on the vector amplitudes
```

Comment:
Thanks for adding this @micmitch! Do you think we can shorten the title? (The whole title is included on the side-bar). What about
```
Plotting: Streamline thickness
=====================

A simple example to vary streamline thickness based on the vector amplitudes 

Author: `@micmitch <https://github.com/micmitch>`
```

Reply:
Done!